index.html

```diff
@@ -0,0 +1,520 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>New York Times</title>
+    <link rel="stylesheet" href="./assets/css/style.css" />
+</head>
+
+<body>
+    <header>
+        <!-- begin nav bar -->
+        <section class="right-div">
+            <button class="subscribe" value="subscribe">subscribe today</button>
+            <button class="login" value="log in">log in</button>
+        </section>
+        <section class="center-div">
+            <img class="nav-title" src="./assets/images/title.png" alt="New York Times">
+        </section>
+        <section class="left-div">
+            <img class="trip" src="./assets/images/trip.png" alt="trip" />
+            <img class="sicon" src="./assets/images/sicon.png" alt="search icon" />
+            <img class="space-cosmos" src="./assets/images/space-cosmos.png" alt="space and cosmos" />
+        </section>
+        <!-- end nav bar -->
+    </header>
+    <hr>
+    <!-- begin main -->
+    <main>
+        <div class="header-img-desc">
+            <img class="header" src="./assets/images/img-header.png" alt="img-header">
+            <img class="main-img" src="./assets/images/first.png" alt="" />
+            <p class="desc">Alan Guth was one of the first physicists to hypothesize the existence of inflation, which
+                explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8
+                billion years ago.<span class="credits">Rick Friedman for The New York Times</span></p>
+        </div>
+        <br>
```

Comment:
No need to put that. If you want more space, you can use `<p>` or css properties like `margin` or `padding`